reused-terms.ttl

```diff
@@ -4,18 +4,32 @@
 @prefix vann: <http://purl.org/vocab/vann/> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix dcatap: <http://data.europa.eu/r5r/> .
+@prefix pink: <https://w3id.org/pink#> .
 
 <https://w3id.org/pink/reused-terms> rdf:type owl:Ontology ;
     owl:versionIRI <https://w3id.org/pink/0.0.1/reused-terms> ;
     dcterms:abstract "Terms from standard vocabularies reused by the PINK Annotation Schema."@en .
 
-dcat:hasVersion rdf:type owl:AnnotationProperty .
+dcat:hasVersion rdf:type owl:AnnotationProperty ;
```

Comment:
I think that it will be a good idea to copy all the annotations from original vocabulary (in this case DCAT or DCAT-AP). To this we can add `pink:conformance` and `vann:usageNote` from the DCAT-AP documentation. 

The rationale with `reused-terms.ttl` is that we want selectively want to pick the concepts we need in PINK Annotation Schema, without blindly importing all of dcterms, dcat, dcat-ap, prov-o, ... This is the same approach as taken by DCAT  and DCAT-AP.